### PR TITLE
Update sa1def.asm

### PR DESCRIPTION
**Pull request by Iwakoro**

### CODE REVIEW

**Comment by Atari2** [comment]
What is this update for?

**Comment by Atari2** [comment]
I think it is better to keep _only_ the defines with the descriptive names instead of the addresses, this way people are forced to use them and code is therefore clearer.

**Comment by Iwakoro** [comment]
To support for "[More Extended Sprites](https://www.smwcentral.net/?p=section&a=details&id=33300)".

File: asm/sa1def.asm
Comment (Iwakoro) [inline asm/sa1def.asm:1]: I forgot to write "|!Base2" at "macro define_exsprite_table."

**Pull request closed without merging**